style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  margin: 0;
+  padding: 0;
+  background: #004882;
+}
+.box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+.box select {
+  width: 250px;
+  height: 50px;
+  font-size: 20px;
+
+  padding: 10px;
+
+  background: #0563af;
+  color: #fff;
+
+  border: none;
+
+  box-shadow: 0 5px 25px rgba(0, 0, 0, 0.5);
+  appearance: none;
+  outline: none;
+}
+.box:before {
+  width: 50px;
+  height: 50px;
+
+  content: "\f358";
+
+  font-family: "Font Awesome 5 Free";
+  font-size: 28px;
+  text-align: center;
+  line-height: 50px;
+  color: #fff;
+
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  background: #007ce0;
+  pointer-events: none;
```

Comment:
What's the use of `pointer-events` ?

Reply:
The ```pointer-events``` property allows you to control whether an element can be the target of pointer (mouse or touch) events.